apps/lwipserver2/include/echo/tuning_params.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+/* Number of buffers used for sending and receiving ethernet frames. */
+#define TX_BUFS 510
+#define RX_BUFS 510
+
+/* Size used for ethernet buffers */
+#define BUF_SIZE 2048
+
+/* Maximum connected TCP clients */
+#define MAX_TCP_CLIENTS 5
+
+/* Size of initial TCP socket reads */
```

Comment:
Can you add a note why exactly this value is chosen? Is it random or related to TCP headers?